front/components/assistant_builder/TagsSuggestDialog.tsx

```diff
@@ -77,6 +82,18 @@ export const TagsSuggestDialog = ({
     }
   }, [isSuggestionsError, sendNotification, setIsOpen]);
 
+  useEffect(() => {
```

Comment:
I don't know how often this will happen to users but while I was selecting tags, it happened to me several times that suggestions get refreshed (revalidated by SWR?) and I lost my `appliedSuggestion`. I cannot come up with a good way would it be worth checking if it has been set once, or users edit appliedSuggestion? 🤔

Reply:
i did not see that, that's weird swr triggers a revalidation without reason ( but not really surprising) . i'll have a look